frontend/src/theme/buttonStyles.tsx

```diff
@@ -0,0 +1,79 @@
+import type { ComponentStyleConfig } from "@chakra-ui/theme";
+import colors from "./colors";
+
+// Button styling
+const Button: ComponentStyleConfig = {
+  baseStyle: {
+    color: colors.white.default,
+    width: "auto",
+  },
+  sizes: {
+    sm: {
+      fontSize: "sm",
+      padding: "8px 12px",
+      borderRadius: "6px",
+    },
+    md: {
+      fontSize: "md",
+      padding: "12px 16px",
+      borderRadius: "8px",
+    },
+  },
+  variants: {
+    primary: {
+      background: colors.blue[400],
```

Comment:
i think you can do it like:
```suggestion
      background: "blue.400",
```

Reply:
do you mind changing all the references to this and removing the `colors` import at the top of the file?